### DISCUSSION TITLE
Delete data registration

### DISCUSSION THREAD

**Discussion by djp4nd4** [Zealot111/OCAP#13] [Q&A]
If I delete a registration file from ocap_web/static/data how can delete it from ocap web interface?
Is there a procedure?
Thanks

**Comment by tekig** [comment]
You can delete a record by changing the data.db file. It can be changed using any sqlite editor, you can use web editors. Before this, it is advisable to make a copy of the file.

You may need a sql query to delete.
Example:
```sql
DELETE FROM operations WHERE id = 1
```

**Comment by djp4nd4** [comment]
Perfect I will try. Thanks for help
